### PR TITLE
Upgrade Spring Cloud and Spring RSA to 1.0.0.RELEASE

### LINKED ISSUE #309: Invalid yaml properties file is not reported in application logs

### PR DESCRIPTION
**Pull request by szpak**
By the way it fixes #309.

Done together with @dst

### CODE REVIEW

**Comment by marcingrzejszczak** [comment]
+1

File: micro-infra-spring-config/src/test/groovy/com/ofg/infrastructure/property/LoadingFromInvalidFileSpec.groovy [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        then:
+            def e = thrown(Exception)
+            e.message.contains("keyForUnquotedCurlyBraces: {cipher}aKey")
+    }
+}
+
+@Configuration
+@EnableAutoConfiguration
+@ComponentScan(basePackageClasses = [ConfigClientAutoConfiguration.class])
+class InvalidApp {
```
Comment (dst) [inline micro-infra-spring-config/src/test/groovy/com/ofg/infrastructure/property/LoadingFromInvalidFileSpec.groovy]: We forgot to rename it to AppWithInvalidConfig

**Comment by dst** [comment]
Great commit :)

File: micro-infra-spring-config/src/test/groovy/com/ofg/infrastructure/property/LoadingFromInvalidFileSpec.groovy (hunk 1 above)
Comment (dst) [inline micro-infra-spring-config/src/test/groovy/com/ofg/infrastructure/property/LoadingFromInvalidFileSpec.groovy]: I have just fixed it.

**Comment by szpak** [comment]
> Great commit :)

Co-author's modesty :).

**Pull request merged**